app/graphql/types/input/library_entry/update_rating_by_id.rb

```diff
@@ -0,0 +1,4 @@
+class Types::Input::LibraryEntry::UpdateRatingById < Types::Input::Base
+  argument :id, ID, required: true
+  argument :rating, Integer, required: true
```

Comment:
Is it possible for arguments to have descriptions (i.e noting its `2-20`)

Reply:
yes we can, I'll add.